src/main/java/in/ac/bits/protocolanalyzer/analyzer/AnalyzerCell.java

```diff
@@ -136,7 +137,12 @@ public void run() {
 			if (!isProcessing) {
 				if (!inputQueue.isEmpty()) {
 					isProcessing = true;
-					process(inputQueue.poll());
+					PacketWrapper temp = inputQueue.poll();
```

Comment:
We were getting the null objects primarily because of the poll() method. The poll() method returns null if there is no object in the queue.